### PR TITLE
(wip) add 'postcss-custom-properties' plugin to webpack-bundled sass files

### PR DESCRIPTION
**Pull request by flootr**
#### Changes proposed in this Pull Request

*

#### Testing instructions

*

Fixes #

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -252,7 +252,7 @@ function getWebpackConfig( { cssFilename, externalizeWordPressPackages = false }
 						{
 							loader: 'postcss-loader',
 							options: {
-								plugins: [ require( 'autoprefixer' ) ],
+								plugins: [ require( 'postcss-custom-properties' )(), require( 'autoprefixer' ) ],
```
Comment (blowery) [inline webpack.config.js]: why the `()` after the require?

Comment (jsnajdr) [reply webpack.config.js]: I'd try this:
1. Load just `postcss-loader` with no options whatsoever. It should pick up a config file.
2. Check if the `.postcssrc.json` config file is loaded. If it's not, try renaming to `postcss.config.js` or some other supported name :)
3. Check that the legacy `node-sass` pipeline is not broken. I suspect that the `build-css:sections` task already is broken.

Comment (flootr) [reply webpack.config.js]: > why the () after the require?

just tried something

**Comment by blowery** [comment]
Different attempt in https://github.com/Automattic/wp-calypso/pull/30162 that appears to work

**Comment by flootr** [comment]
closing in favor of #30162

**Pull request closed without merging**